pkg/db/db.go

```diff
@@ -25,12 +25,24 @@ func db() *sql.DB {
 
 // EnsureLatestVersion ensures that the database is at the latest version by running all migrations.
 func EnsureLatestVersion() {
-	err := goose.SetDialect("sqlite3")
+	driver := config.GetDBDriver()
+	var err error
+	migrationsPath := ""
+
+	switch driver {
```

Comment:
It'd be better to move this `switch` logic somewhere else, in case we want to use it again 😃